src/hal/getconfig_catena4630.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+
+Module:  getconfig_catena4630.cpp
+
+Function:
+        Arduino-LMIC C++ HAL pinmaps for various boards
+
+Copyright & License:
+        See accompanying LICENSE file.
+
+Author:
+        Dhinesh Kumar Pitchai, MCCI       June 2019
+
+*/
+
+#if defined(ARDUINO_MCCI_CATENA_4630) || \
+    /* legacy name */ \
+    defined(ARDUINO_CATENA_4630)
```

Comment:
There is no such legacy name.

Reply:
Yes, This is is not used from 4630, 4617 and 4618. 
@dhineshkumarmcci  Please remove this

Reply:
This is added up by mistake. Sorry, I missed out this in the self review, I will remove it.